### ISSUE TITLE
[Windows] Patch Windows RKE service options

### ISSUE THREAD

**Issue by thxCode** [rancher/rancher#22470]
Seems https://github.com/rancher/rancher/pull/21667 broke the maintaining of RKE Windows service options and system images, we need to patch back to support:
- [ ] Save service options into CRD
- [ ] Output the appropriate images of Windows

**Comment by cjellick** [comment]
You need to provide more details on how to test this.

**Comment by thxCode** [comment]
- For saving service options in CRD:
  1. Fork https://github.com/rancher/kontainer-driver-metadata, modify `k8s_windows_service_options.go` (adjusting the `image-pull-progress-deadline` to `20m` or something else), generate a new data (`go run main.go --write-data=true`).
  1. Setup a master-head rancher server
  1. Modify the rke-metadata-config setting (/v3/settings/rke-metadata-config) to the data.json generated by **step 1**.
   1. Verify the service options have set into `v3/rkek8sserviceoptions` CRD and the spec of CRD is expected, for example:
      ![image](https://user-images.githubusercontent.com/3394724/64501052-6599af00-d2f2-11e9-887b-8cfe0e07494a.png)

- For outputting the appropriate images of Windows, could refer to https://github.com/rancher/rancher/issues/22463:
  1. Review rancher-windows-images.txt
  1. Review rancher-images.txt

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rancher:master-head 768c4bff4

Steps:
- in my fork of metadata, add a new k8s release with custom values for Windows node 
```
kubeletOptions["image-pull-progress-deadline"] = "22m"
```
- run Rancher:master-head  container
- modify the settings to use my fork of the metadata
- refresh the metadata in Rancher, and see the new k8s version available
- add a custom cluster with support for both Linux and Windows node using the new k8s version
- in the Windows node, see the value is applied 
```
PS C:\Users\Administrator> docker inspect kubelet | grep image-pull
            "--image-pull-progress-deadline=22m",
                "--image-pull-progress-deadline=22m",
```
